Podcast/DiscoverViewController.swift

```diff
@@ -21,85 +21,39 @@ class DiscoverViewController: UIViewController, UITableViewDelegate, UITableView
         feedTableView.delegate = self
         feedTableView.dataSource = self
         feedTableView.backgroundColor = .podcastWhiteDark
         feedTableView.separatorStyle = .none
-        feedTableView.register(DiscoverTableViewCell.self, forCellReuseIdentifier: "DiscoverTableViewCellIdentifier")
+        feedTableView.register(FeedTableViewCell.self, forCellReuseIdentifier: "FeedTableViewCellIdentifier")
         view.addSubview(feedTableView)
-        feedTableView.estimatedRowHeight = DiscoverTableViewCell().height
+        feedTableView.rowHeight = FeedTableViewCell().height
         feedTableView.reloadData()
         
-        //topButtons
-        bottomLineView = UIView(frame: CGRect(x: 0, y: topButtonHeight * 2 - lineHeight, width: view.frame.width / 2, height: lineHeight))
-        bottomLineView.backgroundColor = .podcastGreenBlue
-        bottomLineView.layer.cornerRadius = lineHeight / 2
-        view.addSubview(bottomLineView)
-        
-        trendingButton = UIButton(frame: CGRect(x: 0, y: topButtonHeight, width: view.frame.width / 2, height: topButtonHeight))
-        trendingButton.addTarget(self, action: #selector(trendingButtonPress) , for: .touchUpInside)
-        trendingButton.setTitle("Trending", for: .normal)
-        trendingButton.titleLabel!.font = .systemFont(ofSize: 13.0)
-        trendingButton.setTitleColor(.podcastGreenBlue, for: .normal)
-        view.addSubview(trendingButton)
-        
-        categoriesButton = UIButton(frame: CGRect(x: view.frame.width / 2, y: topButtonHeight, width: view.frame.width / 2, height: topButtonHeight))
-        categoriesButton.addTarget(self, action: #selector(categoriesButtonPress), for: .touchUpInside)
-        categoriesButton.setTitle("Categories", for: .normal)
-        categoriesButton.titleLabel!.font = .systemFont(ofSize: 13.0)
-        categoriesButton.setTitleColor(.podcastGreenBlue, for: .normal)
-        view.addSubview(categoriesButton)
-        
-        adjustForScreenSize()
-        
-        
-        let series = Series()
-        series.title = "Planet Money"
         
         //episode static data
-        for i in 0..<4 {
-            let episode = Episode(id: i, title: "Stephen Curry - EP10", dateCreated: Date(), descriptionText: "44 min • In today's show, we visit Buffalo, New York, and get a window into a rough business: Debt collection. This is the story of one guy who tried to make something of himself by getting people to pay their debts. He set up shop in an old karate studio, and called up people who owed money. For a while, he made a good living. And he wasn't the only one in the business—this is also the story of a low-level, semi-legal debt-collection economy that sprang up in Buffalo. And, in a small way, it's the story of the last twenty or so years in global finance, a time when the world went wild for debt.")
-            episode.series = series
-            feedEpisodes.append(episode)
+        for i in 0..<2 {
```

Comment:
this should be moved to a fetchCards() function which we'll use later to make an actual endpoint request